### DISCUSSION TITLE
PKGi PSP v1.1.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/pkgi-psp#26] [Announcements]
### Added

* Show battery level
* .Zip file support
  - Download and extract `.zip` links
* Local .PKG installation
  - Install `.pkg` files from the PSP's memory stick
  - Scan and list packages from `ms0:/PKG`
* Add PSP-Go storage option
  - Edit `config.txt` to change storage location (add line `storage ms0`)

### Fixed

* Fix progress bar ETA when resuming downloads

### Misc

* Add OFW-compatible build (@krazynez)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/pkgi-psp/releases/tag/v1.1.0'>PKGi PSP v1.1.0</a>.</em>

**Comment by 3rdAnd4thRetroGames** [comment]
Thanks for the update!! Looking forward to adding it to my PSP Go.

I did have a question. Is updating as easy as just replacing the PKGi file & running it or do we need to do all the steps again from when we first installed it...? Appreciate the help! & thanks again!

**Reply by TravisDecker** [reply]
I was able to replace the existing files with ones from the update using my PC.

**Reply by bucanero** [reply]
overwriting the `EBOOT.PBP` should be enough to update the app.

**Comment by ghost** [comment]
I'm having some trouble with the download speed, it's only 15kb/s , what am i doing wrong ? The similar app works ok .

**Comment by aleycecy** [comment]
no se cual es el archivo de la tienda

**Comment by bwbuhse** [comment]
Hi, I was wondering how to use the `Install .pkg files from the PSP's memory stick` feature.

When I put a `.pkg` I have into `/PKG`, it shows up in the app but clicking "Download" on it just says it can't make an HTTP connection. Am I doing something wrong?

**Reply by bucanero** [reply]
> Hi, I was wondering how to use the `Install .pkg files from the PSP's memory stick` feature.
> 
> When I put a `.pkg` I have into `/PKG`, it shows up in the app but clicking "Download" on it just says it can't make an HTTP connection. Am I doing something wrong?

It shouldn't be trying to download anything.
Not sure how you're using it. Be sure to change the filter category to `Local PKGs`, is the .pkg listed there?
